pkg/utils/mailbox/mailbox_prom.go

```diff
@@ -0,0 +1,94 @@
+package mailbox
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils"
+)
+
+var mailboxLoad = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	Name: "mailbox_load_percent",
+	Help: "Percent of mailbox capacity used",
+},
+	[]string{"appID", "name", "capacity"},
+)
```

Comment:
Is this our first case of global prom metrics in this module? We might have to be careful with these.

Reply:
Looks like we have some on the monitoring package: https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/monitoring/metrics.go

Reply:
Oh I wasn't expecting that :thinking: 
I'm think that we'll have to undo all these global prom metrics at some point, but I guess it is out of scope for now if it is not in the way.